tests/golden.csv

```diff
@@ -50,54 +50,54 @@ Mike Izbicki,edition.cnn.com,https://edition.cnn.com/2019/12/04/asia/north-korea
 Mike Izbicki,ekurd.net,https://ekurd.net/,,None,,
 Mike Izbicki,ekurd.net,https://ekurd.net/us-president-trump-north-2019-02-27,en-us,2019-02-27 06:30:24.000000 UTC,2019-02-27 18:39:10.000000 UTC,"U.S. President Trump, North Korea leader Kim meet for second summit"
 Mike Izbicki,elcomercio.pe,https://elcomercio.pe/,,None,,
-Mike Izbicki,elcomercio.pe,https://elcomercio.pe/mundo/asia/corea-del-norte-kim-jong-un-ordena-medidas-de-seguridad-ofensivas-antes-de-que-venza-el-ultimatum-a-estados-unidos-noticia/,es-pe,2019-12-30 08:04:33.000000 ,2019-12-30 08:04:52.000000 ,"Corea del Norte: Kim Jong-un ordena medidas de seguridad ""ofensivas"" antes de que venza el ultimátum a Estados Unidos Asia"
+Mike Izbicki,elcomercio.pe,https://elcomercio.pe/mundo/asia/corea-del-norte-kim-jong-un-ordena-medidas-de-seguridad-ofensivas-antes-de-que-venza-el-ultimatum-a-estados-unidos-noticia/,es-pe,04:33.0,04:52.0,"Corea del Norte: Kim Jong-un ordena medidas de seguridad ""ofensivas"" antes de que venza el ultimátum a Estados Unidos Asia"
 Mike Izbicki,elnacional.com.do,https://elnacional.com.do/,,None,,
 Mike Izbicki,elnacional.com.do,https://elnacional.com.do/category/bufeo/,,None,,
 Mike Izbicki,elnacional.com.do,https://elnacional.com.do/category/deportes/,,None,,
-Mike Izbicki,elnacional.com.do,https://elnacional.com.do/corea-del-norte-advierte-que-se-agota-su-paciencia-con-eeuu/,es,2019-10-27 ,,Corea del Norte advierte que se agota su paciencia con EEUU
-Mike Izbicki,elnacional.com.do,https://elnacional.com.do/mala-senal/,es,2020-05-23 ,,Mala señal
-Mike Izbicki,elnacional.com.do,https://elnacional.com.do/zorilla-y-el-pld/,es,2020-05-22 ,,Zorilla y el PLD
+Mike Izbicki,elnacional.com.do,https://elnacional.com.do/corea-del-norte-advierte-que-se-agota-su-paciencia-con-eeuu/,es,10/27/19,,Corea del Norte advierte que se agota su paciencia con EEUU
+Mike Izbicki,elnacional.com.do,https://elnacional.com.do/mala-senal/,es,5/23/20,,Mala señal
+Mike Izbicki,elnacional.com.do,https://elnacional.com.do/zorilla-y-el-pld/,es,5/22/20,,Zorilla y el PLD

```

Comment:
It looks like excel is changing the format of the dates here; you need to make sure they stay in the original format

Reply:
Should I use a different application or manually format it in excel?

Reply:
Personally I use libreoffice instead of excel because I prefer open source software, but you should be able to use excel fine too.  You just have to adjust the settings so that it doesn't change the date format.